### DISCUSSION TITLE
Start application with working directory option.

### DISCUSSION THREAD

**Discussion by Harmen1966** [SafeExamBrowser/seb-win-refactoring#714] [Q&A]
Hi.

Is it possible to start an application with the option using the working directory, and if so how?

Thanks in advance,
Harmen.

**Comment by dbuechel** [comment]
You'd need to be a bit more specific: What would you like to do exactly, and with which application?

**Reply by Harmen1966** [reply]
We want to use an dictionary application in SEB, normally it starts with C:\Program Files\dictionary\dictionary.exe but when I want to use the application in SEB the dictionary language is French in stead of Dutch. I need the option to give the application an working directory so the language is Dutch.

**Reply by dbuechel** [reply]
If the dictionary application supports command-line arguments, then you can specify them in the configuration and potentially pass the language to be used: https://safeexambrowser.org/windows/win_usermanual_en.html#PermittedProcessesSection.

**Comment by Harmen1966** [comment]
Unfortunately, the software has no command line options.
For example, when I use a PowerShell script start-process c:\program files\dictionary\dictionary.exe it's French, but when I use the parameter -WorkingDirectory c:\program files\dictionary then it works fine in Dutch.
________________________________
Van: Damian Büchel ***@***.***>
Verzonden: dinsdag 21 november 2023 15:45
Aan: SafeExamBrowser/seb-win-refactoring ***@***.***>
CC: Harmen Derlagen ***@***.***>; Author ***@***.***>
Onderwerp: Re: [SafeExamBrowser/seb-win-refactoring] Start application with working directory option. (Discussion #714)


If the dictionary application supports command-line arguments, then you can specify them in the configuration and potentially pass the language to be used: https://safeexambrowser.org/windows/win_usermanual_en.html#PermittedProcessesSection.

**Reply by dbuechel** [reply]
Did you try specifying `-WorkingDirectory C:\Program Files\dictionary` as an argument in the SEB configuration?
